.prettierrc.json

```diff
@@ -1,5 +1,6 @@
 {
   "semi": true,
   "tabWidth": 2,
-  "singleQuote": true
+  "singleQuote": true,
+  "trailingComma": "none"
```

Comment:
@kurund my only issue with this is that we end up with unnecessary diffs when commas are added / removed from the last element of a dictionary / list

Reply:
@fatchat 

I think some of the config files didn't have trailing comma and it was added. There was a bit of inconsistency.  Hence, I thought of adding this config. However, you are correct, this will lead to add unnecessary diff in the entire project. let me revert this and add it in config files only.